backend/functions/lib/controllers/cases/cases.controller.test.ts

```diff
@@ -45,126 +45,155 @@ describe('cases controller test', () => {
   });
 
   describe('searchAllCases', () => {
+    const limit = 50;
+    const offset = 0;
+
     test('should return an empty array for no matches', async () => {
       const expected = buildResponseBodySuccess<CaseBasics[]>([], {
         self: mockRequestUrl,
+        limit,
       });
 
       jest.spyOn(CaseManagement.prototype, 'searchCases').mockResolvedValue([]);
 
-      const camsHttpRequest = mockCamsHttpRequest({ query: { caseNumber: '00-00000' } });
-      const actual = await controller.searchAllCases(camsHttpRequest);
+      const camsHttpRequest = mockCamsHttpRequest({
+        method: 'POST',
+        body: { caseNumber: '00-00000', limit, offset },
+      });
+
+      const actual = await controller.searchCases(camsHttpRequest);
       expect(actual).toEqual(expected);
     });
 
     test('should return a next link when result set is larger than limit', async () => {
       const caseNumber = '00-00000';
-      const data = MockData.buildArray(MockData.getCaseBasics, parseInt(limit) + 1);
+      const data = MockData.buildArray(MockData.getCaseBasics, parseInt(limitString) + 1);
```

Comment:
Now that we have `limit`—which is a number—should we just use the following?
```suggestion
      const data = MockData.buildArray(MockData.getCaseBasics, limit + 1);
```

Reply:
For that matter, I'm not even sure we need a variable to store the limit as a string, because it seems like we're only using it in places where we are parsing it to int and using it in a template string.